components/claim-mgt/org.wso2.carbon.claim.mgt.ui/src/main/resources/web/claim-mgt/add-dialect-claim.jsp

```diff
@@ -82,9 +83,9 @@
                         CARBON.showWarningDialog('<fmt:message key="cannot.remove.default.carbon.dialect"/>', null, null);
                         return;
                     }
-                    CARBON.showConfirmationDialog('<fmt:message key="remove.message1"/>' + dialect + '<fmt:message key="remove.message2"/>',
+                    CARBON.showConfirmationDialog('<fmt:message key="remove.message1"/>' + Encode.forJavaScriptAttribute(dialect) + '<fmt:message key="remove.message2"/>',
```

Comment:
You can't directly invoke the java encoding library within the javascript code here. This will not execute the javascript method. We have not looked into encoding javascript within javascript blocks.